silverorange/CombineFirebase
# Support firebase 11

Updates Package.swift to target Firebase 11.2.0. It also updates the package code to remove dependency on FirebaseFirestoreSwift, which has been merged into FirebaseFirestore. See the [Firebase changelog](https://firebase.google.com/support/release-notes/ios#11.2.0) for details on this change.

[ch63843]

###Testing
1. Check out this PR
2. Ensure `CombineFirebase-Package` is the build target
3. Compile the library. There should be no issues aside from a couple deprecation warnings.
